docs/build.md

```diff
@@ -1,67 +1,90 @@
-# Building Larq Compute Engine #
+# Build Larq Compute Engine #
 
 The Larq Compute Engine (LCE) repository consists of two main components:
-LCE for [TensorFlow](#LCE-for-TensorFlow) and LCE for [TensorFlow Lite](#LCE-for-TensorFlow-Lite).
-Each of these components are a collection of optimized ops for TensorFlow and
-TensorFlow Lite.
-These two components can be built indepent of each other. Below we describe
-the build process for each of these components.
-
-### Setup Docker container ###
-We will build the LCE inside a [Docker](https://www.docker.com/) container.
-To be able to build LCE and the LCE converter's
+
+- **LCE Runtime:** which is a collection of highly optimized Ops
```

Comment:
I think tensorflow [writes it lowercase](https://www.tensorflow.org/guide/create_op).

```suggestion
- **LCE Runtime:** which is a collection of highly optimized ops
```

Reply:
I looked into the TF lite guide. they actually use the full name `operator`everywhere. I switch to that in the entire documents then.